README.md

```diff
@@ -1,130 +1,144 @@
+4. **Inference**: Automate deployment of a Nuclio model-serving function.
 
-> **Note:** The demo supports both TensorFlow v1 and v3, there is one (shared) notebook and two code files (one per TF version)
+> **Note:** The demo supports both TensorFlow versions 1 and 2.
+> There's one shared notebook and two code files &mdash; one for each TensorFlow version.
 
 <br><p align="center"><img src="./docs/hvd-flow.png" width="600"/></p><br>
 
 <a id="demo-horovd-image-classification-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
 
 <br><p align="center"><img src="./docs/hvd-pipe.png" width="500"/></p><br>
 
 <a id="demo-face-recognition"></a>
-## [Real-Time Face Recognition with Re-enforced Learning](faces/README.md)
+## Faces Demo &mdash; Real-Time Image Recognition with Deep Learning
 
-Demonstrate real-time face image capture, recognition, and location tracking of identities.
+The [**faces**](faces/README.md) demo demonstrates real-time capture, recognition, and classification of face images over a video stream, as well as location tracking of identities.
 
-This comprehensive demonstration includes multiple components: a live image capture utility, image identification and tracking, a labeling app to tag unidentified faces using Streamlit, and model training.
+This comprehensive demonstration includes multiple components:
+
+- A live image-capture utility.
+- Image identification and tracking using [OpenCV](https://opencv.org/).
+- A labeling application for tagging unidentified faces using [Streamlit](https://www.streamlit.io/).
+- Model training using [PyTorch](https://pytorch.org).
+- Automated model deployment using [Nuclio](https://nuclio.io/)
 
 <a id="demo-face-recognition-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
 
 <br><p align="center"><img src="./faces/workflow.png" width="500"/></p><br>
 
-<a id="demo-sklearn-pipe"></a>
-## Predictive Network/Telemetry Monitoring
-<!-- TODO: When the demo is read, edit the description, and remove the TBD. -->
-
-**TBD under construction**
+<a id="demo-churn"></a>
+## Churn Demo &mdash; Real-Time Customer-Churn Prediction (Telco Customer Churn Data Set)
 
-Demonstrate ingestion of telemetry data from simulator or live stream, feature exploration, data preparation (aggregation), model training, and automated model deployment.
+The [**chrun**](./churn/README.md) demo demonstrates analyses of customer-churn data using the Kaggle [Telco Customer Churn data set](https://www.kaggle.com/blastchar/telco-customer-churn), model training and validation using [XGBoost](https://xgboost.readthedocs.io), and model serving using real-time Nuclio serverless functions.
 
-The demo is maintained in a separate Git repository and also demonstrates how to manage project life cycle using git.
+The demo consists of few MLRun and Nuclio functions and a Kubeflow Pipelines orchestration:
 
-<br><p align="center"><img src="./docs/netops-metrics.png" width="500"/></p><br>
```

Comment:
@Sharon-iguazio why did u remove the image?

Reply:
The image was removed by mistake; I've now returned it (commit 9d79aa4).